crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -676,21 +678,22 @@ fn serialize_call_entry_point(call_entry_point: &CallEntryPoint) -> Vec<Felt252>
         CallType::Delegate => 1,
     };
 
-    let mut output = vec![];
     output.push(Felt252::from(entry_point_type));
     output.push(call_entry_point.entry_point_selector.0.into_());
     output.push(Felt252::from(calldata.len()));
     output.extend(calldata);
     output.push(call_entry_point.storage_address.into_());
     output.push(call_entry_point.caller_address.into_());
     output.push(Felt252::from(call_type));
-
-    output
 }
 
 #[must_use]
 pub fn cheatcode_panic_result(panic_data: Vec<Felt252>) -> Vec<Felt252> {
-    let mut result = vec![Felt252::from(1), Felt252::from(panic_data.len())];
+    let mut result = Vec::with_capacity(panic_data.len() + 2);
```

Comment:
Do we care for such optimisations? I so, are all possible `vec!` replaced?

Reply:
We may not in this case (i don't know), but doing it this way won't hurt us (both speed and readability). If I found a place where `Vec::with_capacity()` can be  easily used, then used it. Of course I could miss something.